src/main/resources/anniv6Resources/localization/eng/Storm/UIstrings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "${ModID}:Storm": {
+    "TEXT": [
+      "Storm",
+      "A thunderstorm is forecasted in this area of the Spire. Lightning will strike either the player or a monster, and you may be rewarded with special charged cards. Make sure to keep your hand out of the rain..."
```

Comment:
Maybe explain what charged does briefly, you don#t have to though if you want to keep it more flavorful